errors/ka.po

```diff
@@ -1,21 +1,21 @@
 msgid ""
 msgstr ""
-"Project-Id-Version: Squid-3\n"
+"Project-Id-Version: Squid-5\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2017-09-16 17:53+1300\n"
-"PO-Revision-Date: 2018-12-30 03:57+0400\n"
-"Last-Translator: George Machitidze <giomac@gmail.com>\n"
-"Language-Team: Squid Developers <squid-dev@squid-cache.org>\n"
+"PO-Revision-Date: 2023-02-15 15:45+0100\n"
+"Last-Translator: Temuri Doghonadze <temuri.doghonadze@gmail.com>\n"
```

Comment:
Please also add your name+email as used here to the first list in the file `errors/TRANSLATORS`.

Reply:
added.